### ISSUE TITLE
Open ReactionPicker on Message LongPress, without opening the context menu first

### ISSUE THREAD

**Issue by mckenziefiege** [GetStream/stream-chat-react-native#108]
We are looking to rewrite the existing functionality of the openReactionSelector in the MessageContent component. Instead of the long press to open the menu and then having to click on “Add Reaction”, we would like to have the longPress immediately open the reaction picker. 

In order to achieve this - we have tried creating a custom component for MessageSimple and MessageContent where our custom MessageSimple component would render the custom MessageContent component. However, it seems to be an endless cycle of having to create custom components (ex: ReactionPicker and MessageReplies) which can not be imported. Is there a way to adjust the openReactionSelector function without having to create custom components?

**Comment by vishalnarkhede** [comment]
Hey @mckenziefiege thanks for bringing this to our attention. Let me get back to you on this one in few hours!!

**Comment by vishalnarkhede** [comment]
Hey @mckenziefiege , we will be publishing a change tomorrow. With that change you will be able to customize the required onLongPress or onPress behaviour as following:

 ```
 import { MessageSimple } from 'stream-chat-react-native' // or 'stream-chat-expo'
 ...
 const MessageUIComponent = (props) => {
  return (
    <MessageSimple
      {...props}
      onLongPress={(thisArg, message, e) => {
        thisArg.openReactionSelector();
        // Similarly, you can also call other methods available on MessageContent
        // component such as handleEdit, handleDelete, showActionSheet
        // Source - https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/MessageSimple/MessageContent.js
      }}
  )
 }
 ```

**Comment by vishalnarkhede** [comment]
Hey @mckenziefiege ,

This has been been published - [v0.6.0](https://github.com/GetStream/stream-chat-react-native/releases/tag/v0.6.0)

Please let us know if it solves your issue!! :)

**Comment by mckenziefiege** [comment]
@vishalnarkhede Works great! Thank you! :)
